internal/mtail/mtail.go

```diff
@@ -299,19 +300,23 @@ func (m *Server) Serve() error {
 		return errors.Errorf("No bind address provided.")
 	}
 	mux := http.NewServeMux()
+	if m.debug {
+		mux.Handle("/", m)
```

Comment:
I think the root handler should remain but we can strip the sensitive information in the template.